mod/community.php

```diff
@@ -107,11 +86,7 @@ function community_content(App $a, $update = 0) {
 
 	$o .= conversation($a,$s,'community',$update);
 
-	if(!get_config('system', 'old_pager')) {
-	        $o .= alt_pager($a,count($r));
-	} else {
-	        $o .= paginate($a);
-	}
+        $o .= alt_pager($a,count($r));
```

Comment:
Sorry to bother you again, line comments don't work well on phone. Here's a new instance of missing space.